spec/controllers/stafftools/assignment_repos_controller_spec.rb

```diff
@@ -5,7 +5,9 @@
   let(:user)    { GitHubFactory.create_owner_classroom_org.users.first }
   let(:student) { GitHubFactory.create_classroom_student               }
 
-  let(:assignment) { create(:assignment, creator: user, organization: user.organizations.first, public_repo: false) }
+  let(:assignment) do
+    Assignment.create(title: 'HTML5', creator: user, organization: user.organizations.first, public_repo: false)
+  end
```

Comment:
@tarebyte This is because I need to specify a title for this assignment, otherwise it will have a different title each time and I can not record the VCR for it.